lib/tower_cli/conf.py

```diff
@@ -37,6 +37,16 @@ def _read(self, fp, fpname):
         """
         # Attempt to read the file using the superclass implementation.
         #
+        # Check the permissions of the file we are considering reading
+        # if the file exists and the permissions expose it to reads from
+        # other users, raise a warning
+        if os.path.isfile(fpname):
+            file_permission = os.stat(fpname)
+            nth_permission = [(file_permission.st_mode/8**i)%8 for i in range(3)]
```

Comment:
It would probably be cleaner/more readable here to explicitly check the masking bits here:

```
if (file_permission.st_mode & os.IRGRP) or (file_permission.st_mode & os.IROTH) ...
```

for example